extensions/smallrye-graphql-client/deployment/pom.xml

```diff
@@ -83,5 +119,29 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>test-keycloak</id>
```

Comment:
We need to make sure that when this profile isn't active, then the two new tests are skipped. Right now, when I run the test suite without `-Dtest-containers`, it fails because it attempts to run all tests and doesn't have the keycloak-related variables. I'm trying to find out how best to do this

Reply:
@sberyozkin do you know how? Even when I exclude the two new tests, the testing framework still seems to pick up the `@QuarkusTestResource` annotation on them and tries to run a Keycloak container

Reply:
I'm not sure whether with basic file-based auth it is possible to set up authentication expiry, probably not.

Reply:
@rubik-cube-man would you be able to move the `DynamicGraphQLClientWebSocketAuthenticationExpiryTest` with its Keycloak setup into a separate module in the `integration-tests` module? Preferably asap, because this should be included in the 3.6 release...  If too hard, I'd say let's add that test later and go without it for now.

Reply:
I will start work on this now!

Reply:
I hadn't really explored this directory until now. I just discovered some tests for WebSockets inside this module as well! Do you think it's worth migrating all my tests over to the integration test module to keep all similar tests in the same location?

Reply:
Thanks!! The module should be skipped  if `test-containers` property isn't defined. See https://github.com/quarkusio/quarkus/blob/main/integration-tests/opentelemetry-jdbc-instrumentation/pom.xml#L293 for an example of an integration-tests module that does the same

Reply:
> Do you think it's worth migrating all my tests over to the integration test module to keep all similar tests in the same location?

No, the point is that we need to separate tests that need Keycloak to run. The other websocket tests don't need Keycloak. Only `DynamicGraphQLClientWebSocketAuthenticationExpiryTest`.

Reply:
I just managed to get it all working in its own module! Is this what you had in mind?
It took a bit longer than expected, but I got there in the end 😌 
I've added it as a native test, as that seemed to be what a few other things also seemed to implement. I've not personally played with Quarkus native, and haven't got a clue how to test it on my own computer. If it doesn't work, or I've done it totally wrong I'm happy to remove that stuff from the tests (or implement it correctly).